pkg/features/features.go

```diff
@@ -18,6 +18,9 @@ const (
 
 	// CustomizedClusterResourceModeling indicates if enable cluster resource custom modeling.
 	CustomizedClusterResourceModeling featuregate.Feature = "CustomizedClusterResourceModeling"
+
+	// PolicyPreemption indicates if high-priority policies could preempt resource templates which are matched by low-priority policies.
+	PolicyPreemption featuregate.Feature = "PolicyPreemption"
```

Comment:
```suggestion
	PolicyPreemption featuregate.Feature = "PropagationPolicyPreemption"
```
As I commented at https://github.com/karmada-io/karmada/pull/3684#discussion_r1259416715, we'd better note the feature gate more clearly.